test/test_modules.py

```diff
@@ -174,6 +174,71 @@ def test_convnet(
     y = convnet(x)
     assert y.shape == torch.Size([*batch, expected_features])
 
+@pytest.mark.parametrize("in_features", [3, 10, None])
+@pytest.mark.parametrize(
+    "input_size, depth, num_cells, kernel_sizes, strides, paddings, expected_features",
+    [(10, None, None, 3, 1, 0, 32 * 4 * 4 * 4), (10, 3, 32, 3, 1, 1, 32 * 10 * 10 * 10)],
```

Comment:
This seems pretty "big" and I wonder how much time it will take to allocate these tensors.
Do you thing we can reduce the size (32 -> 4 or something like it?)

Reply:
it's ok if testing on `test_convnet` is acceptable, cos tensors in test_conv3dnet  are smaller than in test_convnet.